preinstall.js

```diff
@@ -0,0 +1,42 @@
+var exec = require('child_process').exec;
+var spawn = require('child_process').spawn;
+var readline = require('readline');
+var terminal = require('terminal-utilities')
+
+var rl = readline.createInterface({
+	input: process.stdin,
+	output: process.stdout
+});
+
+exec('which wget', function(err,stdout,stderr){
+	if(err){
```

Comment:
there is a problem with the use of if/else here all throughout the code.
if an error is encountered it means that `wget` is not present, and hence must be installed. But the code seems to do something else.